html/js/melpa.js

```diff
@@ -150,7 +151,7 @@
       pkgs.push(new melpa.Package({
         name: name,
         version: version,
-        commit: built.props.commit,
+        commit: props.commit || '',
```

Comment:
I think it needs to be done in here or other places referencing `commit` property. Because `null` would be `"null"` in string concatenation. I didn't follow-up [this](https://github.com/melpa/melpa/pull/4736/files#diff-16a4b1562f252501410d42a063023b9fR410) reference any further. After reading the comment [below](https://github.com/melpa/melpa/pull/4739#issuecomment-299770740) I think it is either not needed or needs to be revised to build a proper URI for commit history page. @purcell what do you think?